Documentation/asagi.rst

```diff
@@ -27,14 +27,20 @@ example on SuperMuc
 
 .. code-block:: bash
 
-   module switch mpi.ibm mpi.intel #since the ibm mpi interferes with python mpi
-   module load python/2.7_anaconda_mpi gcc netcdf/mpi cmake
+   module load mpi
+   module load python/3.5_intel 
+   module load netcdf/mpi
+   module load hdf5/mpi/1.8.18
+   module unload intel
+   module load intel/17.0
```

Comment:
You can use module switch here instead of unload/load.
Not necessary to change but something you might want to know